pages/contact/_form.html

```diff
@@ -12,7 +12,8 @@
     value="New inquiry from {{ site.url }}"
   />
   <input type="hidden" name="_viewing" id="_viewing" value="v1-static" />
-  <input type="hidden" name="_next" value="{{ site.url }}/thanks/" />
+  <input type="hidden" name="_next" id="_next" value="{{ site.url }}/thanks/" />
+  <input type="hidden" name="hello_gruntwork" value=""/>
```

Comment:
We have a "honey pot" configured in FormBucket, but we didn't actually put the form element in our forms, so no bot would ever be tripped up by it...

Reply:
Nice find!